OrbitCore/TracepointEventBuffer.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+#define ORBIT_CORE_TRACEPOINT_EVENT_BUFFER_H_
+
+#include <limits>
+#include <set>
+
+#include "SamplingProfiler.h"
+#include "capture_data.pb.h"
+
+class TracepointEventBuffer {
+ public:
+  TracepointEventBuffer() : max_time_(0), min_time_(std::numeric_limits<uint64_t>::max()) {}
+
+  std::vector<orbit_client_protos::TracepointEventInfo> GetTracepointEvents(
```

Comment:
If we are following `EventBuffer` here, this should be called `GetTracepointEventsInTimeRangeForThreadId`.

Reply:
That method does something else. 'GetTracepointEvents()' maches the method 'GetCallstacks() const' in EventBuffer.

Reply:
Yes, you are right. Thank you!